package.json

```diff
@@ -75,7 +75,7 @@
   },
   "scripts": {
     "ui-assets": "copy-aragon-ui-assets -n aragon-ui ./build",
-    "start": "npm run ui-assets && parcel serve src/index.html --port 3000 --out-dir ./build --no-cache",
+    "start": "npm run ui-assets && parcel serve src/index.html --port $ARAGON_CLIENT_PORT || 3000 --out-dir ./build --no-cache",
```

Comment:
As @izqui pointed out offline, this expression doesn't actually work in Bash. Only test when `$ARAGON_CLIENT_PORT` was set. 

Will update.